### PR TITLE
chore: update ui kit dependency

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,6 @@
     "wire-web-config-default-staging": "https://github.com/wireapp/wire-web-config-default#v0.13.2"
   },
   "devDependencies": {
-    "@types/js-cookie": "2.2.1",
```
Comment (ffflorian) [inline package.json:56]: Why did you remove them?

Comment (AndyLnd) [reply package.json:56]: It was in there twice (line 66)
